calour/database.py

```diff
@@ -316,6 +333,9 @@ def delete_annotation(self, annotation_details):
 
     def remove_feature_from_annotation(self, features, annotation_details):
         '''remove a feature from the annotation in the database (if allowed)
+        This is an optional function for databases that support manual annotations (level L4).
```

Comment:
what is the diff between this func and `delete_annotation`?

Reply:
added explanation. since an annotation can contain many features, delete removes the annotation from all the features it contains, whereas remove_feature_from annotation keeps the annotation but removes the association with the specific feature.